client/src/component/review.jsx

```diff
@@ -74,12 +74,12 @@ const Review = ({currHotelReview, readMore}) => (
           </div>
         </div>
       </div>
-      <div id='disclaimer'>
-        <hr></hr>
-        This response is the subjective opinion of the management representative and not of TripAdvisor LLC.
-        <br></br>
-        <br></br>
-      </div>
+    </div>
+    <hr></hr>
+    <div id='disclaimer'>
+      This response is the subjective opinion of the management representative and not of TripAdvisor LLC.

```

Comment:
Remember we're Trip AdCoba, not Trip Advisor, so you might want to change that name.